### PR TITLE
chore: updated blocklet management features

### PR DESCRIPTION
**Pull request by rahul0208**
Fixed details with Blocklets. 
Removed install/ upgrade marketplace document. Merged it with Blocklets

### CODE REVIEW

File: src/blocklet/environments/index.md [hunk 1]
```diff
@@ -9,11 +9,49 @@ tags:
...
+
+The *Overview* tab provides details like version, status, location, source, etc, for the installed Blocklet binary. It also captures basic usage metrics like uptime, CPU, and memory usage.
+
+![](./images/blocklets-overview.png)
+
+The page also shows different data locations for the different types of files used by the Blocklet.
+- APP Directory: The location to store Blocklet binary.
+- Data Directory: The location to store all data processed by a Blocklet.
+- Log Directory: The location for the logs generated by the Blocklet
+- Cache Directory: The location used for caching all data.
```
Comment (wangshijun) [inline src/blocklet/environments/index.md:36]: The location used for caching all data.

=> The location used for caching data by the blocklet.

File: src/blocklet/environments/index.md [hunk 2]
```diff
@@ -9,11 +9,49 @@ tags:
...
+![](./images/blocklets-access-url.png)
+
+### Configuration Tab
+
+The *Configuration* tab displays different Blocklet and System environment variables. Additionally, you can create user variables for providing Blocklet required parameters.
+
+![](./images/blocklets-config.png)
+
+### Blocklet Variables
+
```
Comment (wangshijun) [inline src/blocklet/environments/index.md:53]: These variables are outdated, please upgrade your node to latest version and update the variables list.

And the variables section need more content, here is a skeleton:

- User variables: defined by blocklet developer can be customized by user to change the behavior of blocklet, this section also need to cover how user can add/delete/change variables, you can change to dev registry and install kitchen-sink-demo blocklet for testing
- Blocklet variables: most of variables within this category are defined by ABT Node, and are immutable, but some of them can be customized.
- System Variables: these are ABT Node related, and are immutable

File: src/blocklet/index.md [hunk 3]
```diff
@@ -1,16 +1,18 @@
...
-This chapter will introduce Blocklet related content,include:
+Previously, you have deployed the ABT Node and have installed Blocklets from the [Marketplace](../marketplace). In this section, you will work with Blocklet administration features available in ABT Node. You will also look at the various types of Blocklets available with ABT Node.
 
-- [Basic Operations](./basic-operations)
-- [Blocklet Environments](./environments)
+## Contents
+
+- [Blocklet Management](./basic-operations)
+- [Configure Blocklet](./environments)
 - [Core Blocklets](./core-blocklets)
```
Comment (wangshijun) [inline src/blocklet/index.md:18]: The core blocklet page needs an update, since some blocklets are deprecated.

**Review by wangshijun — CHANGES_REQUESTED** [review]

**Comment by wangshijun** [comment]
Besides, @rahul0208 please try several demo blocklets form the dev registry to learn more about blocklet management features provided in ABT Node.

There are quire a few critical missing parts from the current docs:

1. Auth Service: Team members, passport, permissions, please try out auth-demo
2. Blocklet interfaces, please try out kitchen-sink-demo blocklet
3. Logs, please checkout the logs tab on sidebar

File: src/blocklet/index.md (hunk 3 above)
Comment (rahul0208) [reply src/blocklet/index.md:18]: May I know how to determine core blocklets as there are many on the market place?

File: src/blocklet/environments/index.md (hunk 2 above)
Comment (rahul0208) [reply src/blocklet/environments/index.md:67]: Updated the details

**Comment by rahul0208** [comment]
@wangshijun  I have mentioned that the blocklets are provided authentication and logging services. The two features are quite detailed and adding them under Blocklet did not felt appropriate. I will add two sections(log and teams) and connect them from this. what do you think?

File: src/blocklet/index.md (hunk 3 above)
Comment (wangshijun) [reply src/blocklet/index.md:18]: Core blocklet can be found https://registry.arcblock.io/

- Block explorer: the universal block explorer that supports query/browse/display data from any blockchain
- ABT Node Launcher: Launch node or service, private beta is not there yet, but should be included
- Certificate Manager: automatically take over SSL certificate generating and renewing within abt node
- FS Chain Manager: get a development chain up and running instantely
- Wallet Playground: show the potential of arcblock platform with demos
- NFT Marketplace: decentralized marketplace for buy and sell NFT
- NFT Store: start your NFT online business with a few clicks

**Comment by wangshijun** [comment]
> @wangshijun I have mentioned that the blocklets are provided authentication and logging services. The two features are quite detailed and adding them under Blocklet did not felt appropriate. I will add two sections(log and teams) and connect them from this. what do you think?

OK, let's add that part in a separate section

File: src/blocklet/index.md (hunk 3 above)
Comment (rahul0208) [reply src/blocklet/index.md:18]: Updated the list of core blocklets.

**Review by wangshijun — APPROVED** [review]

**Pull request merged**